src/App.jsx

```diff
@@ -14,70 +15,85 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () => {
+  const [selectedGood, setSelectedGood] = useState('Jam');
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
+  const handleClick = (item) => {
+    setSelectedGood(item);
+  };
 
-      <button
-        data-cy="ClearButton"
-        type="button"
-        className="delete ml-3"
-      />
-    </h1>
+  const handleGoodSelected = item => (
+    <tr
+      key={item}
+      data-cy="Good"
+      className="has-background-success-light"
+    >
+      <td>
+        <button
+          data-cy="RemoveButton"
+          type="button"
+          className="button is-info"
+          onClick={() => handleClick(null)}
+        >
+          -
+        </button>
+      </td>
 
-    <table className="table">
-      <tbody>
-        <tr data-cy="Good">
-          <td>
-            <button
-              data-cy="AddButton"
-              type="button"
-              className="button"
-            >
-              +
-            </button>
-          </td>
+      <td data-cy="GoodTitle" className="is-vcentered">
+        {item}
+      </td>
+    </tr>
+  );
 
-          <td data-cy="GoodTitle" className="is-vcentered">
-            Dumplings
-          </td>
-        </tr>
+  const goodDefault = item => (
```

Comment:
handleGoodSelected and goodDefault are redundant. They copy each other. Just use conditions at that places where you need to a change value